packages/core/components/InputOrGroup/styles.module.css

```diff
@@ -97,8 +97,7 @@
   border: none;
   border-top: 1px solid var(--puck-color-grey-8);
   margin: 0;
-  padding-bottom: 16px;
-  padding-top: 16px;
+  padding: 16px;
```

Comment:
I _think_ I understand what this is about. Previously we were relying on inheriting (browser-default) styles to provide left and right padding to fieldsets - but since these styles aren't consistent, or could have been overridden, we're now making them explict?

Reply:
Spot on!

Reply:
Noice